django-stubs/db/models/fields/related_descriptors.pyi

```diff
@@ -86,6 +86,19 @@ class ReverseManyToOneDescriptor:
     def __get__(self, instance: Model, cls: Any = ...) -> type[RelatedManager[Any]]: ...
     def __set__(self, instance: Any, value: Any) -> NoReturn: ...
 
+# Fake class, Django defines 'RelatedManager' inside a function body
+class RelatedManager(BaseManager[_M], Generic[_M]):
```

Comment:
You also changed the base class from `Manager` to `BaseManager`.

I suppose it's more correct (managers *usually* derive from `Manager`, but not always)? But this causes several new inspections in my projects. Seems like something that should be mentioned in release notes.